x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/raw_output.tsx

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+// import { flatten } from 'lodash';
```

Comment:
Is this line needed?

Reply:
Updated in https://github.com/elastic/kibana/pull/144629/commits/d92fb308eb4f6ecde47b57de4802d59fc4251277